src/component/handlers/edit/editOnInput.js

```diff
@@ -88,8 +89,41 @@ function editOnInput(editor: DraftEditor): void {
   var domSelection = global.getSelection();
 
   var {anchorNode} = domSelection;
-  if (anchorNode.nodeType !== Node.TEXT_NODE) {
-    return;
+  const isNotTextNode =
```

Comment:
Totally - it's just something we should keep in mind that changed if we run into problems after shipping this